Updated readme for exit rotation (#141)
stakewise/v3-operator: README.md

### Exit signatures rotation

> Exit signatures from the previous section can become invalid if oracles set will changed. For example if some oracle will compomise it's private keys. In this case DAO can trigger exit signature rotation by increasing exit signature version.

Reviewer: ```suggestion
Exit signatures from the previous section can become invalid if the oracles set changes. For example, if oracle's private key gets compromised, the DAO will have to propose an update of the oracles set that will trigger exit signature rotation.
```

> The operator periodically checks active validators for Vault and if some signatures become outdated after rotation request
> release the operator will submit

Reviewer: ```suggestion
The operator periodically checks active validators of the Vault and if some exit signatures become outdated, the operator will submit
```
